src/components/panel/panel.js

```diff
@@ -819,11 +820,16 @@ MdPanelRef.prototype.open = function() {
   // TODO(ErinCoughlan) - Cancel any in-progress actions.
 
   var self = this;
+
+  var onOpenComplete = self._config['onOpenComplete'];
+  var onOpenCompleteScope = self._config['scope'] || self._$rootScope;
```

Comment:
`self._config['scope']` will always be defined. We default it to a new scope off of `$rootScope`.